### DISCUSSION TITLE
Getting notifications to wrong email address

### DISCUSSION THREAD

**Discussion by dprothero** [vercel/community#1639] [Help]
### Summary

I'm getting Vercel notifications to my work email address, which I have associated with the GitHub account I use to login to Vercel. However, in Vercel, I've set my personal email address as my primary email.

How do I get Vercel to send notifications to the correct email address?

**Comment by amyegan** [comment]
Hi David! Vercel [notifications](https://vercel.com/docs/concepts/dashboard-features/notifications#) are delivered via the dashboard and the email set for your account. You can make changes to the type of notifications you want to receive in your [notification settings](https://vercel.com/account/notifications).

Any notifications that come from GitHub, such as comments on PRs, can be configured on your GitHub account. You can find more information about changing GitHub notification settings at https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications

**Reply by dprothero** [reply]
I understand. What I'm saying is I am receiving Vercel notifications (not GitHub) at dprothero@twilio.com, yet the email address set on my Vercel account is dpro@crit.tech.
